### PR TITLE
Improvements to ts->xlsx->ts

### PR DESCRIPTION
**Pull request by s-t-e-f-a-n**
Improved qTsConverter to produce ts-A->xlsx->ts-B with ts-B identical to ts-A and taking into account following improvements:
- included source and sourcelanguage information
- includes xlsx headers and support for: Context, ID, Source, Translation, TranslationType, comment, extracomment, translatorcomment, Location
- added scripts for bulk conversion of ts-files and xlsx-files
- added script to build on Debian / Ubuntu for usage as a cli tool

### CODE REVIEW

File: src/internal/TitleHeaders.hpp [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+		<extracomment>extra comment example</extracomment>
+		<translatorcomment>translator comment example</translatorcomment>
+        <translation>translation example</translation>
+		<translation type="unfinished"></translation>
+		<translation type="vanished">Beispiel1</translation>
+		<translation type="obsolete">Beispiel2</translation>
+    </message>
+</context>
+</TS>
+ */
```
Comment (guerinoni) [inline src/internal/TitleHeaders.hpp]: Since is in README you can remove it from here

**Review by guerinoni — COMMENTED** [review]
Please remove `.vscode` stuff and add a section in readme for documenting all 3 new scripts :)

**Comment by guerinoni** [comment]
Also testing is failing, I suggest to add test for your cases

**Comment by s-t-e-f-a-n** [comment]
Haven't used the tests. Checking now.

**Comment by s-t-e-f-a-n** [comment]
Hey @guerinoni please check again.

**Comment by guerinoni** [comment]
@s-t-e-f-a-n can you format code and check the macos pipeline? :)

**Comment by s-t-e-f-a-n** [comment]
hey @guerinoni, clang-format should be fine now. Don't know what's wrong with the macos pipeline. Is the workflow correct? as the error tells about a deviation of x86_64/arm64: found architecture 'x86_64', required architecture 'arm64'

**Pull request merged**